docs/README.md

```diff
@@ -1,13 +1,14 @@
 
 # Design Docs
+KFService architecture consists of two main specifications, around control pland and data plane. Control plane is responsible for deploying and managing the lifecycle of models, while data plane is responsible for defining the contracts to interact with models once deployed.
```

Comment:
KFServing's architecture consists of two main specifications: control plane and data plane.

Reply:
Your second sentence is somewhat moot with the below bullets.

Reply:
thanks - intentional though. The top level is trying to lay it our in more general sense, or is a language which is more widely understood. 'Managing the lifecycle of deployed models' reaches out to everyone (architects, PMs etc.) , as opposed to 'Defines the specification of the `kfserving` resource'

Reply:
s/KFService/KFServing

Reply:
replace everything with KFService to KFServing @ellis-bigelow ?

Reply:
KFServing refers to the overall project, KFService refers to the specific resource. I was referring to just this instance.

Reply:
Done